packages/contractkit/src/test-utils/AlphabeticSequencer.ts

```diff
@@ -0,0 +1,14 @@
+const Sequencer = require('@jest/test-sequencer').default
+
+class AlphabeticSequencer extends Sequencer {
+  // @ts-ignore
+  sort(tests) {
+    const copyTests = Array.from(tests)
+    return copyTests.sort(
+      // @ts-ignore
```

Comment:
why the `ts-ignore`?

Reply:
> There's a `dev-utils` package that @nambrot created, i though contractkit was already using that, since it contains ganache setup and other test utilities.
> 
> Maybe we should migrate contractkit to use it, and do the changes there. For now, `celocli` use the same setup, and the attestantion service i believe.

Amazing idea, i will open an issue for it.

Reply:
> why the `ts-ignore`?

migrated sequencer back to js, removed the ts-ignore